### ISSUE TITLE
Pytorch 2.0 ebuild

### ISSUE THREAD

**Issue by apcameron** [aclex/pytorch-ebuild#21]
Please add the ebuild for pytorch 2.0

**Comment by aclex** [comment]
Yes, thank you for the request! I have plans on adding it, but haven't started working on it yet. So yes, will increase its priority after your message. Hopefully soon to be done.
